src/useSubscription/models/channel.ts

```diff
@@ -193,6 +193,14 @@ export default class Channel<T extends Action = Action> {
     }
   }
 
+  public refresh(): Promise<void> {
+    this.scope.stop()
+    this.scope = effectScope()
```

Comment:
It's unfortunate vue doesn't have a way to "restart" scope, keeping this property readonly

Reply:
Yeah I also wanted it to work that way.